### ISSUE TITLE
uninitialized constant Middleman::S3SyncExtension::OPTIONS (NameError)

### ISSUE THREAD

**Issue by ajsharp** [fredjean/middleman-s3_sync#90]
Was trying to use the .s3_sync file, but ran into this error during loading.

```
/Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-s3_sync-3.3.0/lib/middleman-s3_sync/extension.rb:73:in `read_config': uninitialized constant Middleman::S3SyncExtension::OPTIONS (NameError)
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-s3_sync-3.3.0/lib/middleman-s3_sync/extension.rb:35:in `after_configuration'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/extension.rb:131:in `block in bind_after_configuration'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/uber-0.0.15/lib/uber/options.rb:96:in `instance_exec'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/uber-0.0.15/lib/uber/options.rb:96:in `proc!'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/uber-0.0.15/lib/uber/options.rb:82:in `evaluate_for'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/uber-0.0.15/lib/uber/options.rb:60:in `call'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/hooks-0.4.1/lib/hooks/hook.rb:59:in `evaluate_callback'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/hooks-0.4.1/lib/hooks/hook.rb:55:in `execute_callback'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/hooks-0.4.1/lib/hooks/hook.rb:40:in `block in run'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/hooks-0.4.1/lib/hooks/hook.rb:39:in `each'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/hooks-0.4.1/lib/hooks/hook.rb:39:in `inject'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/hooks-0.4.1/lib/hooks/hook.rb:39:in `run'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/hooks-0.4.1/lib/hooks.rb:55:in `run_hook_for'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/hooks-0.4.1/lib/hooks/instance_hooks.rb:6:in `run_hook'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/core_extensions/extensions.rb:179:in `initialize'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/core_extensions/data.rb:33:in `initialize'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/application.rb:210:in `initialize'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/core_extensions/request.rb:56:in `new'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/core_extensions/request.rb:56:in `inst'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/cli/build.rb:95:in `shared_instance'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/cli/build.rb:62:in `build'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/cli.rb:72:in `method_missing'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor/command.rb:29:in `run'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor/command.rb:126:in `run'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/lib/middleman-core/cli.rb:20:in `start'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/gems/middleman-core-3.4.0/bin/middleman:18:in `<top (required)>'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/bin/middleman:23:in `load'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/bin/middleman:23:in `<main>'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/bin/ruby_executable_hooks:15:in `eval'
        from /Users/ajsharp/.rvm/gems/ruby-2.1.5/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by fredjean** [comment]
I'll look into this issue later tonight.

**Comment by ajsharp** [comment]
+1 thx for the quick fix. 

- Alex Sharp

**Comment by fredjean** [comment]
You are welcome. It shipped with v3.3.1.

-- Fred

Alex Sharp wrote:
>
> +1 thx for the quick fix.
>
> - Alex Sharp
>
> > On Oct 3, 2015, at 10:05 PM, Frederic Jean
